build-logic/convention/build.gradle.kts

```diff
@@ -40,6 +40,9 @@ dependencies {
     compileOnly(libs.firebase.performance.gradlePlugin)
     compileOnly(libs.kotlin.gradlePlugin)
     compileOnly(libs.ksp.gradlePlugin)
+
+    // These dependency below allows us to get [LibrariesForLibs] the class provided by [libs] in the plugins
+    implementation(files(libs.javaClass.superclass.protectionDomain.codeSource.location))
```

Comment:
This looks like the same hack/workaround as #318 for the underlying issue of https://github.com/gradle/gradle/issues/15383.

Reply:
Right I missed #317 and #318 sorry for that. The motivation behind this was that the workaround is working for quite a time.